src/utils/validators.js

```diff
@@ -803,13 +915,6 @@ const validateDayOfMonth = (month, day) => {
   }
 }
 
-const validateNonFutureDate = (dateStr) => {
```

Comment:
Note: this is not removed, but shifted up + renamed to group similar functions tgt